lib/src/Base/Graph/DrawableImplementation.cxx

```diff
@@ -1062,6 +1064,7 @@ String DrawableImplementation::__repr__() const
       << " legend=" << legend_
       << " data=" << data_
       << " color=" << color_
+	  << " isColorExplicitlySet=" << isColorExplicitlySet_
```

Comment:
There are unnecessary black spaces which make the source code look funny:

![image](https://github.com/openturns/openturns/assets/31351465/b467b4eb-2d18-495f-b590-289c09f81870)

```suggestion
      << " isColorExplicitlySet=" << isColorExplicitlySet_
```